flipperdevices/flipperzero-firmware
# NFC: Custom UID entry when adding manually

# What's new

- Implemented a UID entry field for all the NFC types under the "Add Manually" submenu

# Verification 

- Go to NFC > Add Manually and choose any of the options with the new UID entry field (i.e. not the two generic NFC-A options)
- Enter a UID, and the UID of newly created card should match the entered UID

# Checklist (For Reviewer)
